src/security.md

```diff
@@ -7,17 +7,62 @@ created with it seriously.
 This page describes how to report any vulnerabilities that you find,
 and lists best practices to minimize the risk of introducing a vulnerability.
 
+## Security philosophy
+Flutter security strategy is based on five key pillars
```

Comment:
Should we refer to this as Dart's security strategy here? Or if needed, "Dart and Flutter's"?

Perhaps formatted like this too:
```suggestion
## Security philosophy

Dart's security strategy is based on five key pillars:
```